rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -250,23 +250,36 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 		{
 		case Qt::Key_E:
 		{
+			if (i < 0)
+			{
+				return;
+			}
+
```

Comment:
Pc can be moved here or not?

Reply:
it can but I thought that if something else might need it in the future the decleration can remain as is.

Reply:
My main concern is that `i` isn't guarded with the `i < 0` in the outer scope, which might produce a regression if someone is careless in the future.

Reply:
checking it aways caused a weird effect where you had to click on a random instruction to do any action such as opening the regsters editor or capturing SPU image which was weird... it needs to be checked where it matters not where its not.

Reply:
hence moving the pc to where it matters after the i check

Reply:
Made it so it never needs to be checked, just defaults to GetPc() if condition is not met.